mslib/msui/mscolab.py

```diff
@@ -1027,9 +1027,16 @@ def add_operation(self):
             self.error_dialog.showMessage('Path can\'t contain spaces or special characters')
             return
 
+        user_template = MSUIDefaultConfig.new_flighttrack_template
+        xml_waypoints = "".join(
```

Comment:
that goes wrong. we need the coordinates of the waypoints.

the same problem is solved for the local flighttrack.